tests/testthat/test-accept-reject-submission.R

```diff
@@ -0,0 +1,30 @@
+context("accept-reject-submission.R")
+
+test_that("accept_submission makes the correct string", {
+  m <- mock(TRUE, FALSE)
```

Comment:
This works, but some of us were a little confused about the fact that there were different values here and what they represent. It might be a little easier to understand if we used one value and cycled it:

```r
  m <- mock(TRUE, cycle = TRUE)
```

Reply:
Reasoning was simply to see that it was calling the function twice and returning the expected values (i.e. it should return `TRUE` on the first call and `FALSE` if called a second time). This could definitely just be a cycle (and I will probably change it to be that way in a different check). Will leave this one as it is for now since it is a good reference for me on how it works.